src/darwin/Framework/CHIP/MTRDevicePairingDelegate.h

```diff
@@ -19,24 +19,24 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
-typedef NS_ENUM(NSUInteger, CHIPPairingStatus) {
-    kSecurePairingSuccess = 0,
-    kSecurePairingFailed,
-    kUnknownStatus,
+typedef NS_ENUM(NSUInteger, MTRPairingStatus) {
+    MTRPairingStatusUnknown = 0,
+    MTRPairingStatusSuccess = 1,
+    MTRPairingStatusFailed = 2
```

Comment:
Shouldn't constant names start with `k`?  That's what I see for things like `kSecAttrKeyClass` and whatnot...  Similar for the constants that describe the MTRDevice structured-graph output, by the way.

Reply:
No, they should not. Those is old carbon style things. All these should have prefixes, this is the normal pattern.